src/components/share-overlay/_share-overlay.scss

```diff
@@ -58,7 +58,7 @@
         position: absolute;
         width: 32px;
       }
-      .check-icon {
+      .icon-check {
```

Comment:
Why those changes?

Reply:
because it either changing it here, or changing it in the share component

Reply:
Why to change it at all?

Reply:
that's the structure of all the icons in the icon component. 
<i className={[style.icon, style.iconCheck].join(' ')}/>);